src/hotspot/share/classfile/vmIntrinsics.cpp

```diff
@@ -316,6 +314,10 @@ bool vmIntrinsics::disabled_by_jvm_flags(vmIntrinsics::ID id) {
   case vmIntrinsics::_fmaF:
     if (!InlineMathNatives || !UseFMA) return true;
     break;
+  case vmIntrinsics::_dtanh:
+  case vmIntrinsics::_dcbrt:
+    if (!InlineMathNatives || !InlineIntrinsics) return true;
```

Comment:
You also need to add check for UseLibmIntrinsic here:
#ifdef AMD64
    if (!UseLibmIntrinsic)  return true;
#endif